solutionbox/inception/setup.py

```diff
@@ -25,6 +25,9 @@
     'datalab_solutions',
     'datalab_solutions.inception',
   ],
+  # setup.py needs to be deployed so it can be repackaged from local installation for cloud run.
+  data_files=[('/datalab/packages_setup/inception', ['setup.py'])],
```

Comment:
It can be relative, but AFAIK it is relative to sys.prefix (which also depends on what user it runs under). This should work with Linux/Mac, if user installs it with root.